### PR TITLE
Various minor things

### PR DESCRIPTION
**Pull request by gsmet**
Hey there,

I wanted to know what you were up to and went through the (amazing!) documentation to understand things better. I fixed a few very minor things along the way.

- [Make sure ITs dependencies are properly constrained](https://github.com/quarkiverse/quarkus-langchain4j/commit/ee88f876f1e999a4afc7ad8ddf081fb5d0e60af5)
- [Minor adjustments to the documentation](https://github.com/quarkiverse/quarkus-langchain4j/commit/b5fcec2cbe1f6d0209118d8cbd1aa3efb6cf1d3e) 

  - Avoid Quarkus prefix and double extension name in UI (this was discussed with Holly a long time ago but I fixed it in the template only yesterday)
  - Various very minor improvements/fixes

### CODE REVIEW

**Comment by gsmet** [comment]
BTW, should we publish the documentation to the Antora website already or do you want to wait for the first release before doing so?

**Comment by geoand** [comment]
All the credits go to @cescoffier for the docs!

**Comment by geoand** [comment]
I would prefer we wait for the first release - likely to happen today

**Review by geoand — APPROVED** [review]
Thanks a lot for this!

File: docs/modules/ROOT/pages/llms.adoc [hunk 1]
```diff
@@ -16,11 +16,11 @@ Continued research and development in LLMs are constantly pushing the boundaries
...
 Note that each LLM has a different feature set.
 Please check the specific documentation for the LLM you are using to see what features are available:
 
-- link:../openai.adoc[OpenAI (GT-3/4)]
-- link:../hugging-face.adoc[Hugging Face]
-- link:../ollama.adoc[Ollama]
\ No newline at end of file
+- xref:openai.adoc[OpenAI (GT-3/4)]
+- xref:hugging-face.adoc[Hugging Face]
+- xref:ollama.adoc[Ollama]
```
Comment (gsmet) [inline docs/modules/ROOT/pages/llms.adoc:26]: @cescoffier this is the only really important change: make sure you use `xref:` for your links. I suspect the `link:../` would have been broken.

**Pull request merged**